spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -491,12 +478,106 @@ class Recurrent[T : ClassTag](var batchNormParams: BatchNormParams[T] = null)
 }
 
 object Recurrent extends ContainerSerializable {
+
+  private val batchDim = 1
```

Comment:
Seems duplicated in both object and class.

Reply:
changed